hnn_core/network.py

```diff
@@ -309,10 +326,10 @@ def _instantiate_feeds(self, n_trials=1):
                                 params=p_unique[src_type],
                                 gid=gid)
                         )
-                event_times_per_source.update({src_type: event_times})
+                feed_times[src_type].append(event_times)
 
             # list of dict of list of list
-            self.trial_event_times.append(event_times_per_source.copy())
+            self.feed_times = feed_times
```

Comment:
Pretty sure this is indented one too deep. Loop over trials appends each trial to `feed_times[src_type]`, after which `self.feed_times` stores the whole dictionary.